pkg/controller/woopra_test.go

```diff
@@ -0,0 +1,62 @@
+package controller
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/codeready-toolchain/registration-service/test"
+
+	"github.com/gin-gonic/gin"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type TestWoopraSuite struct {
+	test.UnitTestSuite
+}
+
+func TestRunWoopraSuite(t *testing.T) {
+	suite.Run(t, &TestAuthConfigSuite{test.UnitTestSuite{}})
+}
+
+func (s *TestAuthConfigSuite) TestWoopraHandler() {
+	// Create a request to pass to our handler. We don't have any query parameters for now, so we'll
+	// pass 'nil' as the third parameter.
+	req, err := http.NewRequest(http.MethodGet, "/api/v1/woopra", nil)
```

Comment:
shouldn't be the endpoint `/woopra-domain`?

Reply:
yeah, good catch, fixed here: fd6538e88bf12f20ad946a8f8d04acebd5acbb02